code/modules/language/mushroom.dm

```diff
@@ -0,0 +1,11 @@
+/datum/language/mushroom
+	name = "Mushroom"
+	desc = "A language that consists of the sound of periodic gusts of spore-filled air being released."
+	speech_verb = "puffs"
+	ask_verb = "puffs inquisitively"
+	exclaim_verb = "poofs loudly"
+	whisper_verb = "puffs quietly"
+	key = "y"
```

Comment:
🚲 🏠 but it should be m(ushroom) or p(ff)

Reply:
>:(

Reply:
y :^)